.pre-commit-hooks.yaml

```
@@ -52,3 +52,9 @@
   description: Validate for PR Template file presence
   entry: hooks/scripts/check-prtemplate.sh
   language: script
+
+- id: check-print-keyword
+  name: Check for Print keyword
+  description: Validate for print keyword in the code
+  entry: hooks/scripts/check-print-keyword.sh
+  language: script
```

should we limit the files to be scanned to python instead of defaulting to everything ?

```suggestion
- id: check-print-keyword
  name: Check for Print keyword
  description: Validate for print keyword in the code
  entry: hooks/scripts/check-print-keyword.sh
  files: \.py$
  language: script
```

---

for this one I am not very sure, should we target .py files or all the files because in some of the yaml we do have inline lambda code.

---

Yep, that is possible lambda code might be having print statements as well.